packages/wonder-blocks-button/components/button.js

```diff
@@ -11,6 +11,7 @@ export type SharedProps = {|
      */
     children: string,
 
+    // TODO: incorporate
```

Comment:
🤷‍♂️ It may also make sense to just comment out this prop since it doesn't do anything anyway!